.nuspec

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="utf-8"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2013/05/nuspec.xsd">
+    <metadata>
+        <id>MesserliInformatikAg.ServerCommunication</id>
+        <version>0.1.0</version>
+        <authors>Jan Nils Ferner, Dominic Groth, Thomas Bruderer</authors>
+        <owners>messerli-informatik-ag</owners>
+        <projectUrl>http://github.com/messerli-informatik-ag/utility</projectUrl>
+        <license type="expression">MIT or Apache-2.0</license>
+        <description>Rest request library</description>

```

Comment:
"Classes dealing with HTTP communication and the retrieval and deserialization of resources."

Reply:
^ Use this string in the readme aswell